package.json

```diff
@@ -4,12 +4,15 @@
   "version": "1.0.0",
   "description": "Networking sockets for Electron apps",
   "license": "MIT",
-  "repository": "github:foxglove/electron-socket",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/foxglove/electron-socket.git"
+  },
   "author": {
-    "name": "Foxglove",
-    "email": "contact@foxglove.dev",
-    "url": "https://foxglove.dev/"
+    "name": "Foxglove Technologies",
```

Comment:
I would either say `Foxglove` (the colloquial name) or `Foxglove Technologies Inc` (the legal company name since `Foxglove Inc` was taken), but `Foxglove Technologies` on its own isn't a term we use